<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
     MIDI Virtual Piano Keyboard
     Copyright (C) 2008-2016, Pedro Lopez-Cabanillas

     This program is free software; you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
     the Free Software Foundation; either version 3 of the License, or
     (at your option) any later version.

     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.

     You should have received a copy of the GNU General Public License along
     with this program; If not, see http://www.gnu.org/licenses/
 </comment>
 <class>MacSynthSettingsDialog</class>
 <widget class="QDialog" name="MacSynthSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>182</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>SoundFont:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>soundfont_dls</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="soundfont_dls">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btn_soundfont">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>256</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCheckBox" name="reverb_dls">
     <property name="text">
      <string>Use Internal Reverb</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="default_dls">
     <property name="text">
      <string>Default Apple DLS Sound Set</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>soundfont_dls</tabstop>
  <tabstop>btn_soundfont</tabstop>
  <tabstop>default_dls</tabstop>
  <tabstop>reverb_dls</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>default_dls</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundfont_dls</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>default_dls</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_soundfont</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>default_dls</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MacSynthSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MacSynthSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
